Code/Framework/AzQtComponents/AzQtComponents/Utilities/Conversions.cpp

```diff
@@ -39,15 +39,15 @@ namespace AzQtComponents
         return AZ::Color(static_cast<float>(rgb.redF()), static_cast<float>(rgb.greenF()), static_cast<float>(rgb.blueF()), static_cast<float>(rgb.alphaF()));
     }
 
-    QString toString(double value, int numDecimals, const QLocale& locale, bool showGroupSeparator)
+    QString toString(double value, int numDecimals, const QLocale& locale, bool showGroupSeparator, bool truncate)
```

Comment:
suggestion: It might be useful to have a couple of quick sanity tests for this function to ensure we get the expected value. They could be really simple functions with no fixture, there should hopefully be a place for them in `AzQtComponents` I'd have thought. Could be a separate PR but would be great to capture the expected behavior of this function given various inputs

Reply:
Done😀